homeassistant/components/tts/amazon_polly.py

```diff
@@ -148,6 +146,10 @@ def supported_options(self):
 
     def get_tts_audio(self, message, language=None, options=None):
         """Request TTS file from Polly."""
+        if language not in self.all_languages:
```

Comment:
Ohhh, reading your newer comment ("What is to do") clarified it for me, will do!